frappe/desk/page/setup_wizard/setup_wizard.css

```diff
@@ -113,8 +105,13 @@
 .setup-wizard-slide .frappe-control[data-fieldtype="Attach Image"] {
 	width: 140px;
 	height: 180px; /*depends on presence of heading*/
+	margin-top: 20px;
 	text-align: center;
-	margin-left: calc((100% - 140px)/2);
+}
+
+/*image control should be centred on profile slide*/
+.slide-wrapper[data-slide-id="2"] .frappe-control[data-fieldtype="Attach Image"] {
```

Comment:
As a layout object does all of the positioning for controls (coarse tune-able via sections and columns), and classes are not assignable to them on the go, it would require some slight breaking to control a control's (he he) alignment :)

Will get on with it.